shared/chat/conversation/list-area/normal/index.stories.tsx

```diff
@@ -24,7 +24,7 @@ const makeMoreOrdinals = (
   num = __STORYSHOT__ ? 10 : 100
 ): I.List<Types.Ordinal> => {
   if (direction === 'prepend') {
-    const oldStart = ordinals.size ? Types.ordinalToNumber(ordinals.first()) : firstOrdinal
+    const oldStart = ordinals.size ? Types.ordinalToNumber(ordinals.first() as Types.Ordinal) : firstOrdinal
```

Comment:
is this working around a bug or ignoring the fact that `ordinals` could be empty?

Reply:
TS doesn't understand that checking size means first() is non null